tests/test_grpc.py

```diff
@@ -93,6 +93,32 @@ def test_download_missing_file(mapdl, tmpdir):
         mapdl.download('__notafile__', target)
 
 
+def test_grpc_custom_ip():
+    from ansys.mapdl.core import launch_mapdl
+    
+    ip='127.0.0.2'
```

Comment:
```suggestion
    ip = '127.0.0.2'
```
Add whitespace around certain operators:
> Always surround these binary operators with a single space on either side: assignment ( = ), augmented assignment ( += , -= etc.), comparisons ( == , < , > , != , <> , <= , >= , in , not in , is , is not ), Booleans ( and , or , not ).

Reply:
Noted. Changing that those lines in the next commit.